### PR TITLE
Blog post - Podman + Systemd

### PR DESCRIPTION
**Pull request by EmilienM**
This blog post explains how we can manage containers from Systemd.

### CODE REVIEW

**Comment by EmilienM** [comment]
@rhatdan @TomSweeneyRedHat for review please!

**Comment by ghost** [comment]
LGTM - let's see if @TomSweeneyRedHat or @rhatdan got something to add and then we can merge.

File: _posts/2018-09-13-systemd.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+By setting up a Systemd unit file on the host, we can have the host
+automatically start, stop, check the status, and otherwise manage a container
+as a regular systemd service. 
+
+
+If you're running Enforced SElinux mode, you will need to enable this bool so
+containers can be run with Systemd:
+```shell
+sudo setsebool -P container_manage_cgroup on
```
Comment (rhatdan) [inline _posts/2018-09-13-systemd.md]: This is only required if you are running systemd as pid 1 inside of a container?

File: _posts/2018-09-13-systemd.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+podman pull docker.io/redis
+podman run -d --name redis -p 6379:6379 redis
+```
+
+Check that the container is actually running with `podman ps`:
+```
+CONTAINER ID   IMAGE                            COMMAND                  CREATED          STATUS             PORTS                    NAMES
+411a6c6be7d8   docker.io/library/redis:latest   docker-entrypoint.s...   10 minutes ago   Up 5 minutes ago   0.0.0.0:6379->6379/tcp   redis
+```
+
```
Comment (rhatdan) [inline _posts/2018-09-13-systemd.md:37]: Perhaps show that redis is running in the  container using 
podman top?

And 
You should probably stop the container.
podman stop redis

File: _posts/2018-09-13-systemd.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+Podman wasn't designed to manage containers start-up order, dependency
+checking or failed containers recovery.
+In fact, this job can be done by external tools and this blog post describes
+how we can use the systemd initialization service to work with Podman
+containers.
+
+<!--readmore-->
+
+Thanks to Systemd, Podman and Docker formatted containers can be managed in
```
Comment (rhatdan) [inline _posts/2018-09-13-systemd.md]: Not sure what this sentence is saying.

Are you saying podman can be used as a replacement for Docker?
 
Or are you trying to say that Podman can manage Docker images?  Which I would call OCI Images of just Container Images.

File: _posts/2018-09-13-systemd.md (hunk 1 above)
Comment (EmilienM) [reply _posts/2018-09-13-systemd.md]: Right, I'll drop it.

File: _posts/2018-09-13-systemd.md (hunk 2 above)
Comment (EmilienM) [reply _posts/2018-09-13-systemd.md:37]: Ok I'll show that redis is running in the container.
However for "podman stop" I added a note later in the post.

**Comment by EmilienM** [comment]
Pull-request updated, HEAD is now dee37888352810ac6f05ec11ee5a8d0e2534fe7e

File: _posts/2018-09-13-systemd.md (hunk 3 above)
Comment (EmilienM) [reply _posts/2018-09-13-systemd.md]: I replaced "Podman and Docker formatted containers" by containers, to make it simple.

**Comment by rhatdan** [comment]
LGTM
@TomSweeneyRedHat PTAL

**Pull request merged**